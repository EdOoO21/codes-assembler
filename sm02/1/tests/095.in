89 52 986 508 104 604 916 770 -738 939 11 -942 -715 650 -974 231 -827 291 -434 653 -604 -909 163 478 83 246 -602 288 328 -777 -263 -426 522 629 -208 -235 -685 -634 -622 -198 955 -305 271 -633 230 -188 219 -481 42 348 -676 -773 505 -800 399 -263 790 317 -38 -374 -971 -951 -243 409 85 87 210 -282 -387 899 -240 -595 -119 248 847 -221 -749 836 -729 601 913 -833 153 -940 -971 837 -562 341 -203 -526 56 -38 699 511 344 174 -662
