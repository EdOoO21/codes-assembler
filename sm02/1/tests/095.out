-662
174
344
511
699
-38
56
-526
-203
341
-562
837
-971
-940
153
-833
913
601
-729
836
-749
-221
847
248
-119
-595
-240
899
-387
-282
210
87
85
409
-243
-951
-971
-374
-38
317
790
-263
399
-800
505
-773
-676
348
42
-481
219
-188
230
-633
271
-305
955
-198
-622
-634
-685
-235
-208
629
522
-426
-263
-777
328
288
-602
246
83
478
163
-909
-604
653
-434
291
-827
231
-974
650
-715
-942
11
939
-738
770
916
604
104
508
986
52
89
